packages/api-explorer/src/AuthBox.jsx

```diff
@@ -10,7 +10,7 @@ function Securities({ authInputRef, operation, onChange, oauth, auth, onSubmit }
     const securities = securityTypes[type];
     return (
       <form key={type} onSubmit={onSubmit}>
-        <h3>{type} Auth</h3>
+        <h3>{`${type} Auth`}</h3>
```

Comment:
Weird change...

Reply:
It was complaining about this as a React object needing to be on its own line, so it was trying to convert this into:

```
<h3>
  {type}
  {' '}
  Auth
</h3>
```

Silly.